app/views/transactions/_form.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="d-flex justify-content-center">
+  <%= form_with(model: [@category, @transaction]) do |form| %>
+    <% if transaction.errors.any? %>
+      <div class="text-danger">
+        <h2>Error: <%= pluralize(transaction.errors.count, "error") %></h2>
+        <ul>
+          <% transaction.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <div class="my-2">
+      <%= form.label :name %>
+      <%= form.text_field :name %>
+    </div>
+
+    <div class="my-2">
+      <%= form.label :amount %>
+      <%= form.number_field :amount %>
+    </div>
+
+    <div class="d-flex justify-content-center my-4">
+      <div class="submit-btn">
+        <%= form.submit %>
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] As required when creating a transaction:
The user fills out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)

Your app only takes 2 inputs and assumes the category is the one selected when going to the transactions page. Kindly fix this.